Presentation.Web/Controllers/API/V1/OData/ItContractOverviewReadModelsController.cs

```diff
@@ -32,5 +38,47 @@ public IHttpActionResult Get([FromODataUri] int organizationId, int? responsible
 
             return query.Match(onSuccess: Ok, onFailure: FromOperationError);
         }
+
+        /// <summary>
+        /// V2 style OData endpoint suited for consumption by clients using UUID's for entity identity
+        /// </summary>
+        /// <param name="organizationUuid"></param>
+        /// <param name="responsibleOrganizationUnitUuid"></param>
+        /// <returns></returns>
+        [EnableQuery]
+        [SwaggerResponse(HttpStatusCode.OK, type: typeof(ODataListResponse<ItContractOverviewReadModel>))]
+        [ODataRoute("ItContractOverviewReadModels")]
+        public IHttpActionResult GetByUuid(Guid organizationUuid, Guid? responsibleOrganizationUnitUuid = null)
+        {
+            var orgDbId = _identityResolver.ResolveDbId<Organization>(organizationUuid);
+            if (orgDbId.IsNone)
+            {
+                return FromOperationError(new OperationError("Invalid org id", OperationFailure.NotFound));
+            }
+
+            int? orgUnitId = null;
+            if (responsibleOrganizationUnitUuid.HasValue)
+            {
+                var unitDbId = _identityResolver.ResolveDbId<OrganizationUnit>(responsibleOrganizationUnitUuid.Value);
+                if (unitDbId.IsNone)
+                {
+                    return FromOperationError(new OperationError("Invalid org unit id", OperationFailure.BadInput));
+                }
+
+                orgUnitId = unitDbId.Value;
+            }
+
+            return GetOverviewReadModels(orgDbId.Value, orgUnitId);
+        }
+
+        private IHttpActionResult GetOverviewReadModels(int organizationId, int? responsibleOrganizationUnitId)
+        {
+            var query = responsibleOrganizationUnitId == null
+                ? _readModelsService.GetByOrganizationId(organizationId)
+                : _readModelsService.GetByOrganizationIdAndIdOrgOrganizationUnitSubTree(organizationId,
```

Comment:
Small naming comment: this looks like a tough method to name, but when just skimming the name I have a hard time figuring out what "AndIdOrgOrganizationUnit" covers